webui/react/src/pages/Settings/UserSettings.settings.ts

```diff
@@ -0,0 +1,81 @@
+import { any } from 'io-ts';
+
+import { SettingsConfig } from 'hooks/useSettings';
+
+type Char =
+  | 'A'
+  | 'B'
+  | 'C'
+  | 'D'
+  | 'E'
+  | 'F'
+  | 'G'
+  | 'H'
+  | 'I'
+  | 'J'
+  | 'K'
+  | 'L'
+  | 'M'
+  | 'N'
+  | 'O'
+  | 'P'
+  | 'Q'
+  | 'R'
+  | 'S'
+  | 'T'
+  | 'U'
+  | 'V'
+  | 'W'
+  | 'X'
+  | 'Y'
+  | 'Z';
+
+interface KeyboardShortcut {
+  ctrl: boolean;
+  meta: boolean;
+  alt: boolean;
+  shift: boolean;
+  letter?: Char;
```

Comment:
Keyboard shortcuts don't have to use only capital alphabetic characters. I'd recommend looking around for a good data representation of keyboard shortcuts to copy instead of doing it from scratch.

Reply:
Just to clarify, this set up can have lowercase alphabetic characters since 'u' and 'U' both have `code=KeyU`.

Maybe I was searching the wrong keywords.. but I wasn't able to find any packages that can properly represent keys. Most related packages out there like [react-hotkeys](https://www.npmjs.com/package/react-hotkeys) or [react-hotkeys-hook](https://github.com/JohannesKlauss/react-hotkeys-hook) only focus on the key representation in code.

Reply:
Shortcuts could also use `,` or `[`, etc.

Reply:
MDN has a list of key codes: https://developer.mozilla.org/en-US/docs/web/api/ui_events/keyboard_event_code_values